src/smclarify/bias/metrics/common.py

```diff
@@ -122,10 +122,13 @@ def series_datatype(data: pd.Series, values: Optional[List[str]] = None) -> Data
     # if datatype is boolean or categorical we return data as categorical
     data_type = DataType.CATEGORICAL
     data_uniqueness_fraction = divide(data.nunique(), data.count())
-    logger.info(f"data uniqueness fraction: {data_uniqueness_fraction}")
     # Assumption: user will give single value for threshold currently
     # Todo: fix me if multiple thresholds for facet or label are supported
     if data.dtype.name == "category" or (isinstance(values, list) and len(values) > 1):
+        logger.info(
+            f"Column {data.name} with data uniqueness fraction {data_uniqueness_fraction} is classifed as a "
+            f"{data_type.name} column"
+        )
         return data_type
     if data.dtype.name in ["str", "string", "object"]:
```

Comment:
I'd almost prefer not returning data_type here and replacing the following `if` by `elif`.

Like
```
if data.dtype.name == "category" or (isinstance(values, list) and len(values) > 1):
    pass
elif data.dtype.name in ["str", "string", "object"]:
   ...
logger.info(...)
return data_type
```
It's fine to leave as is, though.